java/fury-core/src/main/java/org/apache/fury/ThreadLocalFury.java

```diff
@@ -257,6 +257,9 @@ public void setClassLoader(ClassLoader classLoader) {
 
   @Override
   public void setClassLoader(ClassLoader classLoader, StagingType stagingType) {
+    if (classLoader == null) {
```

Comment:
This should be redundant, there is already such a way in `org.apache.fury.util.LoaderBinding#setClassLoader`

Reply:
Good catch